src/main/java/com/iexec/worker/compute/ComputeManagerService.java

```diff
@@ -34,13 +34,39 @@
 import org.springframework.stereotype.Service;
 
 import java.io.File;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
 
 
 @Slf4j
 @Service
 public class ComputeManagerService {
 
     private static final String STDOUT_FILENAME = "stdout.txt";
+    /**
+     * Examples of timeout depending on category:
+     * <ul>
+     *     <li>XS:   50 * 0.02 = 1 minute</li>
```

Comment:
Putting couple examples is a good idea, but why not taking real world ones instead? https://github.com/iExecBlockchainComputing/PoCo/blob/v5.3.0/config/config.json
`iexec category show 0 --chain bellecour`

Reply:
I think they are real ones.
`iexec category show 0 --chain bellecour` gives a `workClockTimeRef` of 300. That means `maxExecutionTime` is 3000 seconds = 50 minutes. Other calculations should also be based on real world values.
Does it look strange to you?

Reply:
Oh yes you are right :eyeglasses: !